test: Intermittent issue in             feature_init.py", line 88, in run_test                                        with node.wait_for_debug_log([terminate_line]):                       AssertionError: [node 0] Expected messages "[b'scheduler thread start']" does not partially match log:

https://drahtbot.space/temp_scratch/feature_init_55.tar.zstd

```
 test  2024-04-26T22:21:51.801000Z TestFramework.node0 (DEBUG): Node stopped 
 test  2024-04-26T22:21:51.801000Z TestFramework (INFO): Starting node and will exit after line b'Validating signatures for all blocks' 
 test  2024-04-26T22:21:51.809000Z TestFramework.node0 (DEBUG): bitcoind started, waiting for RPC to come up 
 node0 2024-04-26T22:22:36.237116Z [init] [init/common.cpp:155] [LogPackageVersion] Bitcoin Core version v27.99.0-256e1703197f-dirty (release build) 
 node0 2024-04-26T22:22:36.413060Z [init] [init.cpp:729] [InitParameterInteraction] InitParameterInteraction: parameter interaction: -bind set -> setting -listen=1 
 node0 2024-04-26T22:22:36.728701Z [init] [node/chainstatemanager_args.cpp:54] [ApplyArgsManOptions] Script verification uses 3 additional threads 
 node0 2024-04-26T22:22:39.242708Z [init] [kernel/context.cpp:20] [Context] Using the 'sse4(1way),sse41(4way),avx2(8way)' SHA256 implementation 
 node0 2024-04-26T22:22:39.246556Z [init] [random.cpp:105] [ReportHardwareRand] Using RdSeed as an additional entropy source 
 node0 2024-04-26T22:22:39.247534Z [init] [random.cpp:108] [ReportHardwareRand] Using RdRand as an additional entropy source 
 node0 2024-04-26T22:22:44.674789Z [init] [init/common.cpp:126] [StartLogging] Default data directory /root/.bitcoin 
 node0 2024-04-26T22:22:44.676675Z [init] [init/common.cpp:127] [StartLogging] Using data directory /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest 
 node0 2024-04-26T22:22:44.678508Z [init] [init/common.cpp:132] [StartLogging] Config file: /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/bitcoin.conf 
 node0 2024-04-26T22:22:44.683316Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: regtest="1" 
 node0 2024-04-26T22:22:44.685354Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] bind="127.0.0.1" 
 node0 2024-04-26T22:22:44.685644Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] connect="0" 
 node0 2024-04-26T22:22:44.685827Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] deprecatedrpc="create_bdb" 
 node0 2024-04-26T22:22:44.685990Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] discover="0" 
 node0 2024-04-26T22:22:44.686152Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] dnsseed="0" 
 node0 2024-04-26T22:22:44.686316Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] fallbackfee="0.0002" 
 node0 2024-04-26T22:22:44.686477Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] fixedseeds="0" 
 node0 2024-04-26T22:22:44.686639Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] keypool="1" 
 node0 2024-04-26T22:22:44.686799Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] listenonion="0" 
 node0 2024-04-26T22:22:44.686961Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] natpmp="0" 
 node0 2024-04-26T22:22:44.687125Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] peertimeout="999999999" 
 node0 2024-04-26T22:22:44.687294Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] port="11660" 
 node0 2024-04-26T22:22:44.687462Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] printtoconsole="0" 
 node0 2024-04-26T22:22:44.687776Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] rpcdoccheck="1" 
 node0 2024-04-26T22:22:44.687949Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] rpcport="16660" 
 node0 2024-04-26T22:22:44.688177Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] rpcservertimeout="99000" 
 node0 2024-04-26T22:22:44.688350Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] server="1" 
 node0 2024-04-26T22:22:44.688538Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] shrinkdebugfile="0" 
 node0 2024-04-26T22:22:44.688711Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] unsafesqlitesync="1" 
 node0 2024-04-26T22:22:44.688872Z [init] [common/args.cpp:815] [logArgsPrefix] Config file arg: [regtest] upnp="0" 
 node0 2024-04-26T22:22:44.689462Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: blockfilterindex="1" 
 node0 2024-04-26T22:22:44.689665Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: coinstatsindex="1" 
 node0 2024-04-26T22:22:44.689845Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: datadir="/ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0" 
 node0 2024-04-26T22:22:44.690016Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: debug="" 
 node0 2024-04-26T22:22:44.762080Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: debugexclude="libevent" 
 node0 2024-04-26T22:22:44.762679Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: debugexclude="leveldb" 
 node0 2024-04-26T22:22:44.762873Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: debugexclude="rand" 
 node0 2024-04-26T22:22:44.763052Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: loglevel="trace" 
 node0 2024-04-26T22:22:44.763227Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: logsourcelocations="" 
 node0 2024-04-26T22:22:44.763407Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: logthreadnames="" 
 node0 2024-04-26T22:22:44.763699Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: logtimemicros="" 
 node0 2024-04-26T22:22:44.764028Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: txindex="1" 
 node0 2024-04-26T22:22:44.764257Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: uacomment="testnode0" 
 node0 2024-04-26T22:22:44.764484Z [init] [common/args.cpp:815] [logArgsPrefix] Command-line arg: v2transport="0" 
 node0 2024-04-26T22:22:44.766868Z [init] [init.cpp:1137] [AppInitMain] Using at most 125 automatic connections (1048564 file descriptors available) 
 node0 2024-04-26T22:22:45.168354Z [init] [script/sigcache.cpp:104] [InitSignatureCache] Using 16 MiB out of 16 MiB requested for signature cache, able to store 524288 elements 
 node0 2024-04-26T22:22:45.569644Z [init] [validation.cpp:1946] [InitScriptExecutionCache] Using 16 MiB out of 16 MiB requested for script execution cache, able to store 524288 elements 
 node0 2024-04-26T22:22:46.401599Z [scheduler] [util/thread.cpp:20] [TraceThread] scheduler thread start 
 node0 2024-04-26T22:23:07.846802Z [init] [httpserver.cpp:239] [InitHTTPAllowList] [http] Allowing HTTP connections from: 127.0.0.0/8 ::1/128 
 node0 2024-04-26T22:23:07.953388Z [init] [httpserver.cpp:385] [HTTPBindAddresses] Binding RPC on address ::1 port 16660 
 node0 2024-04-26T22:23:08.045702Z [init] [httpserver.cpp:385] [HTTPBindAddresses] Binding RPC on address 127.0.0.1 port 16660 
 node0 2024-04-26T22:23:08.049891Z [init] [httpserver.cpp:464] [InitHTTPServer] [http] Initialized HTTP server 
 node0 2024-04-26T22:23:08.052526Z [init] [httpserver.cpp:466] [InitHTTPServer] [http] creating work queue of depth 16 
 node0 2024-04-26T22:23:08.128420Z [init] [rpc/server.cpp:298] [StartRPC] [rpc] Starting RPC 
 node0 2024-04-26T22:23:08.153880Z [init] [httprpc.cpp:298] [StartHTTPRPC] [rpc] Starting HTTP RPC server 
 node0 2024-04-26T22:23:08.229284Z [init] [httprpc.cpp:247] [InitRPCAuthentication] Using random cookie authentication. 
 node0 2024-04-26T22:23:08.241283Z [init] [rpc/request.cpp:111] [GenerateAuthCookie] Generated RPC authentication cookie /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/.cookie 
 node0 2024-04-26T22:23:08.249290Z [init] [httpserver.cpp:746] [RegisterHTTPHandler] [http] Registering HTTP handler for / (exactmatch 1) 
 node0 2024-04-26T22:23:08.255490Z [init] [httpserver.cpp:746] [RegisterHTTPHandler] [http] Registering HTTP handler for /wallet/ (exactmatch 0) 
 node0 2024-04-26T22:23:08.328687Z [init] [httpserver.cpp:489] [StartHTTPServer] Starting HTTP server with 4 worker threads 
 node0 2024-04-26T22:23:08.535388Z [http] [httpserver.cpp:352] [ThreadHTTP] [http] Entering http event loop 
 node0 2024-04-26T22:23:09.254419Z [init] [wallet/load.cpp:51] [VerifyWallets] Using wallet directory /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest 
 node0 2024-04-26T22:23:09.339693Z [init] [noui.cpp:57] [noui_InitMessage] init message: Verifying wallet(s)… 
 node0 2024-04-26T22:23:09.443386Z [init] [init.cpp:1252] [AppInitMain] Using /16 prefix for IP bucketing 
 node0 2024-04-26T22:23:09.446670Z [init] [noui.cpp:57] [noui_InitMessage] init message: Loading P2P addresses… 
 node0 2024-04-26T22:23:09.948146Z [init] [logging/timer.h:58] [Log] [addrman] CheckAddrman: new 0, tried 0, total 0 started 
 node0 2024-04-26T22:23:09.958508Z [init] [logging/timer.h:58] [Log] [addrman] CheckAddrman: completed (4.41ms) 
 node0 2024-04-26T22:23:09.965912Z [init] [addrdb.cpp:204] [LoadAddrman] Loaded 0 addresses from peers.dat  116ms 
 node0 2024-04-26T22:23:10.072167Z [init] [noui.cpp:57] [noui_InitMessage] init message: Loading banlist… 
 node0 2024-04-26T22:23:10.156877Z [init] [banman.cpp:40] [LoadBanlist] [net] Loaded 0 banned node addresses/subnets  82ms 
 node0 2024-04-26T22:23:10.168432Z [init] [net.cpp:2324] [SetTryNewOutboundPeer] [net] setting try another outbound peer=false 
 node0 2024-04-26T22:23:10.174820Z [init] [net.cpp:3088] [SetNetworkActive] SetNetworkActive: true 
 node0 2024-04-26T22:23:10.563190Z [init] [policy/fees.cpp:470] [Read] [estimatefee] Reading estimates: 190 buckets counting confirms up to 48 blocks 
 node0 2024-04-26T22:23:10.571112Z [init] [policy/fees.cpp:470] [Read] [estimatefee] Reading estimates: 190 buckets counting confirms up to 12 blocks 
 node0 2024-04-26T22:23:10.765095Z [init] [policy/fees.cpp:470] [Read] [estimatefee] Reading estimates: 190 buckets counting confirms up to 1008 blocks 
 node0 2024-04-26T22:23:11.387350Z [init] [node/chainstatemanager_args.cpp:54] [ApplyArgsManOptions] Script verification uses 3 additional threads 
 node0 2024-04-26T22:23:11.566713Z [init] [init.cpp:1507] [AppInitMain] Cache configuration: 
 node0 2024-04-26T22:23:11.570673Z [init] [init.cpp:1508] [AppInitMain] * Using 2.0 MiB for block index database 
 node0 2024-04-26T22:23:11.579309Z [init] [init.cpp:1510] [AppInitMain] * Using 56.0 MiB for transaction index database 
 node0 2024-04-26T22:23:11.582817Z [init] [init.cpp:1514] [AppInitMain] * Using 49.0 MiB for basic block filter index database 
 node0 2024-04-26T22:23:11.584593Z [init] [init.cpp:1516] [AppInitMain] * Using 8.0 MiB for chain state database 
 node0 2024-04-26T22:23:11.671987Z [init] [init.cpp:1535] [AppInitMain] * Using 335.0 MiB for in-memory UTXO set (plus up to 286.1 MiB of unused mempool space) 
 node0 2024-04-26T22:23:12.589231Z [init] [noui.cpp:57] [noui_InitMessage] init message: Loading block index… 
 test  2024-04-26T22:23:12.592000Z TestFramework (DEBUG): Terminating node after terminate line was found 
 node0 2024-04-26T22:23:12.592533Z [init] [node/chainstate.cpp:168] [LoadChainstate] Validating signatures for all blocks. 
 test  2024-04-26T22:23:12.594000Z TestFramework (INFO): Starting node and will exit after line b'scheduler thread start' 
 test  2024-04-26T22:23:12.594000Z TestFramework.utils (DEBUG): Deleting leftover cookie file 
 node0 2024-04-26T22:23:12.596917Z [init] [node/chainstate.cpp:170] [LoadChainstate] Setting nMinimumChainWork=0000000000000000000000000000000000000000000000000000000000000000 
 test  2024-04-26T22:23:12.673000Z TestFramework.node0 (DEBUG): bitcoind started, waiting for RPC to come up 
 node0 2024-04-26T22:23:12.794665Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/blocks/index 
 node0 2024-04-26T22:23:14.096480Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully 
 node0 2024-04-26T22:23:14.199611Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/blocks/index: 0000000000000000 
 node0 2024-04-26T22:23:15.095256Z [init] [node/blockstorage.cpp:508] [LoadBlockIndexDB] LoadBlockIndexDB: last block file = 0 
 node0 2024-04-26T22:23:15.108785Z [init] [node/blockstorage.cpp:512] [LoadBlockIndexDB] LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=201, size=52791, heights=0...200, time=2011-02-02...2024-04-26) 
 node0 2024-04-26T22:23:15.112460Z [init] [node/blockstorage.cpp:523] [LoadBlockIndexDB] Checking all blk files are present... 
 node0 2024-04-26T22:23:15.696764Z [init] [node/chainstate.cpp:108] [CompleteChainstateInitialization] Initializing chainstate Chainstate [ibd] @ height -1 (null) 
 node0 2024-04-26T22:23:15.701181Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate 
 node0 2024-04-26T22:23:15.721779Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully 
 node0 2024-04-26T22:23:15.797646Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate: c1778325e486f12b 
 node0 2024-04-26T22:23:15.911337Z [init] [validation.cpp:4505] [LoadChainTip] Loaded best chain: hashBestChain=0cc8a9d7de2003283477a44a77eb05262bd023e4bfdcbcf60f16d41b40f44124 height=200 date=2024-04-26T22:21:45Z progress=1.000000 
 node0 2024-04-26T22:23:16.108155Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate 
 node0 2024-04-26T22:23:16.114952Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully 
 node0 2024-04-26T22:23:16.116161Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate: c1778325e486f12b 
 node0 2024-04-26T22:23:16.121733Z [init] [validation.cpp:5353] [ResizeCoinsCaches] [Chainstate [ibd] @ height 200 (0cc8a9d7de2003283477a44a77eb05262bd023e4bfdcbcf60f16d41b40f44124)] resized coinsdb cache to 8.0 MiB 
 node0 2024-04-26T22:23:16.123534Z [init] [validation.cpp:5355] [ResizeCoinsCaches] [Chainstate [ibd] @ height 200 (0cc8a9d7de2003283477a44a77eb05262bd023e4bfdcbcf60f16d41b40f44124)] resized coinstip cache to 335.0 MiB 
 node0 2024-04-26T22:23:16.314274Z [init] [noui.cpp:57] [noui_InitMessage] init message: Verifying blocks… 
 node0 2024-04-26T22:23:16.418000Z [init] [validation.cpp:4537] [VerifyDB] Verifying last 6 blocks at level 3 
 node0 2024-04-26T22:23:16.420888Z [init] [validation.cpp:4546] [VerifyDB] Verification progress: 0% 
 node0 2024-04-26T22:23:17.027624Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 16% 
 node0 2024-04-26T22:23:17.204475Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 33% 
 node0 2024-04-26T22:23:17.206795Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 50% 
 node0 2024-04-26T22:23:17.210459Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 66% 
 node0 2024-04-26T22:23:17.212161Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 83% 
 node0 2024-04-26T22:23:17.213658Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 99% 
 node0 2024-04-26T22:23:17.214556Z [init] [validation.cpp:4648] [VerifyDB] Verification: No coin database inconsistencies in last 6 blocks (6 transactions) 
 node0 2024-04-26T22:23:17.226943Z [init] [init.cpp:1596] [AppInitMain]  block index            4635ms 
 node0 2024-04-26T22:23:17.327923Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/txindex 
 node0 2024-04-26T22:23:17.408898Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully 
 node0 2024-04-26T22:23:17.409600Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/txindex: 0000000000000000 
 node0 2024-04-26T22:23:17.420619Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/blockfilter/basic/db 
 node0 2024-04-26T22:23:17.423503Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully 
 node0 2024-04-26T22:23:17.423888Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/blockfilter/basic/db: 0000000000000000 
 node0 2024-04-26T22:23:17.706267Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/coinstats/db 
 node0 2024-04-26T22:23:17.708974Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully 
 node0 2024-04-26T22:23:17.709325Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/coinstats/db: 0000000000000000 
 node0 2024-04-26T22:23:17.722339Z [init] [init.cpp:1682] [AppInitMain] Setting NODE_NETWORK on non-prune mode 
 node0 2024-04-26T22:23:18.110919Z [init] [init.cpp:1791] [AppInitMain] block tree size = 201 
 node0 2024-04-26T22:23:18.112656Z [initload] [util/thread.cpp:20] [TraceThread] initload thread start 
 node0 2024-04-26T22:23:18.118579Z [init] [init.cpp:1804] [AppInitMain] nBestHeight = 200 
 node0 2024-04-26T22:23:18.516667Z [init] [net.cpp:3024] [BindListenPort] Bound to 127.0.0.1:11660 
 node0 2024-04-26T22:23:18.525282Z [init] [net.cpp:3024] [BindListenPort] Bound to 127.0.0.1:18445 
 node0 2024-04-26T22:23:18.527543Z [init] [noui.cpp:57] [noui_InitMessage] init message: Starting network threads… 
 node0 2024-04-26T22:23:18.941750Z [init] [net.cpp:3237] [Start] DNS seeding disabled 
 node0 2024-04-26T22:23:19.342627Z [net] [util/thread.cpp:20] [TraceThread] net thread start 
 node0 2024-04-26T22:23:19.736691Z [txindex] [util/thread.cpp:20] [TraceThread] txindex thread start 
 node0 2024-04-26T22:23:20.454074Z [init] [noui.cpp:57] [noui_InitMessage] init message: Done loading 
 node0 2024-04-26T22:23:20.637899Z [basic block filter index] [util/thread.cpp:20] [TraceThread] basic block filter index thread start 
 node0 2024-04-26T22:23:20.638758Z [msghand] [util/thread.cpp:20] [TraceThread] msghand thread start 
 node0 2024-04-26T22:23:20.655212Z [addcon] [util/thread.cpp:20] [TraceThread] addcon thread start 
 node0 2024-04-26T22:23:20.751656Z [coinstatsindex] [util/thread.cpp:20] [TraceThread] coinstatsindex thread start 
 node0 2024-04-26T22:23:21.049821Z [txindex] [index/base.cpp:195] [Sync] Syncing txindex with block chain from height 0 
 node0 2024-04-26T22:23:21.147000Z [initload] [kernel/mempool_persist.cpp:73] [LoadMempool] Loading 0 mempool transactions from file... 
 node0 2024-04-26T22:23:21.155535Z [initload] [kernel/mempool_persist.cpp:145] [LoadMempool] Imported mempool transactions from file: 0 succeeded, 0 failed, 0 expired, 0 already there, 0 waiting for initial broadcast 
 node0 2024-04-26T22:23:21.336281Z [initload] [util/thread.cpp:22] [TraceThread] initload thread exit 
 node0 2024-04-26T22:23:21.344912Z [basic block filter index] [flatfile.cpp:69] [Allocate] [validation] Pre-allocating up to position 0x100000 in fltr00000.dat 
 node0 2024-04-26T22:23:21.358499Z [basic block filter index] [index/base.cpp:195] [Sync] Syncing basic block filter index with block chain from height 0 
 node0 2024-04-26T22:23:22.965573Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 0 
 node0 2024-04-26T22:23:24.073588Z [txindex] [index/base.cpp:209] [Sync] txindex is enabled at height 200 
 node0 2024-04-26T22:23:24.074095Z [txindex] [util/thread.cpp:22] [TraceThread] txindex thread exit 
 node0 2024-04-26T22:23:25.467454Z [basic block filter index] [index/base.cpp:209] [Sync] basic block filter index is enabled at height 200 
 node0 2024-04-26T22:23:25.467705Z [basic block filter index] [util/thread.cpp:22] [TraceThread] basic block filter index thread exit 
 node0 2024-04-26T22:23:53.661723Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 17 
 node0 2024-04-26T22:24:05.820494Z [scheduler] [net.cpp:2329] [StartExtraBlockRelayPeers] [net] enabling extra block-relay-only peers 
 node0 2024-04-26T22:24:24.630070Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 35 
 node0 2024-04-26T22:24:55.720035Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 54 
 node0 2024-04-26T22:25:26.457788Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 77 
 node0 2024-04-26T22:25:57.224169Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 96 
 node0 2024-04-26T22:26:28.650838Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 117 
 node0 2024-04-26T22:27:00.044150Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 137 
 node0 2024-04-26T22:27:30.385389Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 152 
 node0 2024-04-26T22:28:01.495158Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 168 
 node0 2024-04-26T22:28:31.759504Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 181 
 node0 2024-04-26T22:29:03.312048Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 199 
 node0 2024-04-26T22:29:05.924504Z [coinstatsindex] [index/base.cpp:209] [Sync] coinstatsindex is enabled at height 200 
 node0 2024-04-26T22:29:05.924873Z [coinstatsindex] [util/thread.cpp:22] [TraceThread] coinstatsindex thread exit 
 node0 2024-04-26T22:38:20.253388Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  202ms 
 node0 2024-04-26T22:53:20.271084Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms 
 node0 2024-04-26T23:08:20.330833Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms 
 node0 2024-04-26T23:23:11.404762Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat. 
 node0 2024-04-26T23:23:20.372298Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  16ms 
 node0 2024-04-26T23:38:20.399361Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms 
 node0 2024-04-26T23:53:20.408714Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms 
 node0 2024-04-27T00:08:20.435724Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms 
 node0 2024-04-27T00:23:11.582356Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat. 
 node0 2024-04-27T00:23:20.449989Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms 
 node0 2024-04-27T00:38:20.457850Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms 
 node0 2024-04-27T00:53:20.470015Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms 
 node0 2024-04-27T01:08:20.482908Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  11ms 
 node0 2024-04-27T01:23:11.670237Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat. 
 node0 2024-04-27T01:23:20.492458Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms 
 node0 2024-04-27T01:38:20.502842Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms 
 node0 2024-04-27T01:53:20.512615Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms 
 node0 2024-04-27T02:08:20.523052Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms 
 node0 2024-04-27T02:23:11.771345Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat. 
 node0 2024-04-27T02:23:20.534687Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  10ms 
 node0 2024-04-27T02:38:20.545271Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms 
 node0 2024-04-27T02:53:20.557320Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  10ms 
 test  2024-04-27T03:03:12.596000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "/ci_container_base/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 132, in main
                                       self.run_test()
                                     File "/ci_container_base/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/feature_init.py", line 88, in run_test
                                       with node.wait_for_debug_log([terminate_line]):
                                     File "/usr/lib/python3.12/contextlib.py", line 144, in __exit__
                                       next(self.gen)
                                     File "/ci_container_base/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_node.py", line 524, in wait_for_debug_log
                                       self._raise_assertion_error(
                                     File "/ci_container_base/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_node.py", line 188, in _raise_assertion_error
                                       raise AssertionError(self._node_msg(msg))
                                   AssertionError: [node 0] Expected messages "[b'scheduler thread start']" does not partially match log:
                                    - 2024-04-26T22:23:12.596917Z [init] [node/chainstate.cpp:170] [LoadChainstate] Setting nMinimumChainWork=0000000000000000000000000000000000000000000000000000000000000000
                                    - 2024-04-26T22:23:12.794665Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/blocks/index
                                    - 2024-04-26T22:23:14.096480Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
                                    - 2024-04-26T22:23:14.199611Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/blocks/index: 0000000000000000
                                    - 2024-04-26T22:23:15.095256Z [init] [node/blockstorage.cpp:508] [LoadBlockIndexDB] LoadBlockIndexDB: last block file = 0
                                    - 2024-04-26T22:23:15.108785Z [init] [node/blockstorage.cpp:512] [LoadBlockIndexDB] LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=201, size=52791, heights=0...200, time=2011-02-02...2024-04-26)
                                    - 2024-04-26T22:23:15.112460Z [init] [node/blockstorage.cpp:523] [LoadBlockIndexDB] Checking all blk files are present...
                                    - 2024-04-26T22:23:15.696764Z [init] [node/chainstate.cpp:108] [CompleteChainstateInitialization] Initializing chainstate Chainstate [ibd] @ height -1 (null)
                                    - 2024-04-26T22:23:15.701181Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate
                                    - 2024-04-26T22:23:15.721779Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
                                    - 2024-04-26T22:23:15.797646Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate: c1778325e486f12b
                                    - 2024-04-26T22:23:15.911337Z [init] [validation.cpp:4505] [LoadChainTip] Loaded best chain: hashBestChain=0cc8a9d7de2003283477a44a77eb05262bd023e4bfdcbcf60f16d41b40f44124 height=200 date=2024-04-26T22:21:45Z progress=1.000000
                                    - 2024-04-26T22:23:16.108155Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate
                                    - 2024-04-26T22:23:16.114952Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
                                    - 2024-04-26T22:23:16.116161Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/chainstate: c1778325e486f12b
                                    - 2024-04-26T22:23:16.121733Z [init] [validation.cpp:5353] [ResizeCoinsCaches] [Chainstate [ibd] @ height 200 (0cc8a9d7de2003283477a44a77eb05262bd023e4bfdcbcf60f16d41b40f44124)] resized coinsdb cache to 8.0 MiB
                                    - 2024-04-26T22:23:16.123534Z [init] [validation.cpp:5355] [ResizeCoinsCaches] [Chainstate [ibd] @ height 200 (0cc8a9d7de2003283477a44a77eb05262bd023e4bfdcbcf60f16d41b40f44124)] resized coinstip cache to 335.0 MiB
                                    - 2024-04-26T22:23:16.314274Z [init] [noui.cpp:57] [noui_InitMessage] init message: Verifying blocks…
                                    - 2024-04-26T22:23:16.418000Z [init] [validation.cpp:4537] [VerifyDB] Verifying last 6 blocks at level 3
                                    - 2024-04-26T22:23:16.420888Z [init] [validation.cpp:4546] [VerifyDB] Verification progress: 0%
                                    - 2024-04-26T22:23:17.027624Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 16%
                                    - 2024-04-26T22:23:17.204475Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 33%
                                    - 2024-04-26T22:23:17.206795Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 50%
                                    - 2024-04-26T22:23:17.210459Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 66%
                                    - 2024-04-26T22:23:17.212161Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 83%
                                    - 2024-04-26T22:23:17.213658Z [init] [validation.cpp:4554] [VerifyDB] Verification progress: 99%
                                    - 2024-04-26T22:23:17.214556Z [init] [validation.cpp:4648] [VerifyDB] Verification: No coin database inconsistencies in last 6 blocks (6 transactions)
                                    - 2024-04-26T22:23:17.226943Z [init] [init.cpp:1596] [AppInitMain]  block index            4635ms
                                    - 2024-04-26T22:23:17.327923Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/txindex
                                    - 2024-04-26T22:23:17.408898Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
                                    - 2024-04-26T22:23:17.409600Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/txindex: 0000000000000000
                                    - 2024-04-26T22:23:17.420619Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/blockfilter/basic/db
                                    - 2024-04-26T22:23:17.423503Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
                                    - 2024-04-26T22:23:17.423888Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/blockfilter/basic/db: 0000000000000000
                                    - 2024-04-26T22:23:17.706267Z [init] [dbwrapper.cpp:241] [CDBWrapper] Opening LevelDB in /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/coinstats/db
                                    - 2024-04-26T22:23:17.708974Z [init] [dbwrapper.cpp:249] [CDBWrapper] Opened LevelDB successfully
                                    - 2024-04-26T22:23:17.709325Z [init] [dbwrapper.cpp:274] [CDBWrapper] Using obfuscation key for /ci_container_base/ci/scratch/test_runner/test_runner_₿_🏃_20240426_151850/feature_init_55/node0/regtest/indexes/coinstats/db: 0000000000000000
                                    - 2024-04-26T22:23:17.722339Z [init] [init.cpp:1682] [AppInitMain] Setting NODE_NETWORK on non-prune mode
                                    - 2024-04-26T22:23:18.110919Z [init] [init.cpp:1791] [AppInitMain] block tree size = 201
                                    - 2024-04-26T22:23:18.112656Z [initload] [util/thread.cpp:20] [TraceThread] initload thread start
                                    - 2024-04-26T22:23:18.118579Z [init] [init.cpp:1804] [AppInitMain] nBestHeight = 200
                                    - 2024-04-26T22:23:18.516667Z [init] [net.cpp:3024] [BindListenPort] Bound to 127.0.0.1:11660
                                    - 2024-04-26T22:23:18.525282Z [init] [net.cpp:3024] [BindListenPort] Bound to 127.0.0.1:18445
                                    - 2024-04-26T22:23:18.527543Z [init] [noui.cpp:57] [noui_InitMessage] init message: Starting network threads…
                                    - 2024-04-26T22:23:18.941750Z [init] [net.cpp:3237] [Start] DNS seeding disabled
                                    - 2024-04-26T22:23:19.342627Z [net] [util/thread.cpp:20] [TraceThread] net thread start
                                    - 2024-04-26T22:23:19.736691Z [txindex] [util/thread.cpp:20] [TraceThread] txindex thread start
                                    - 2024-04-26T22:23:20.454074Z [init] [noui.cpp:57] [noui_InitMessage] init message: Done loading
                                    - 2024-04-26T22:23:20.637899Z [basic block filter index] [util/thread.cpp:20] [TraceThread] basic block filter index thread start
                                    - 2024-04-26T22:23:20.638758Z [msghand] [util/thread.cpp:20] [TraceThread] msghand thread start
                                    - 2024-04-26T22:23:20.655212Z [addcon] [util/thread.cpp:20] [TraceThread] addcon thread start
                                    - 2024-04-26T22:23:20.751656Z [coinstatsindex] [util/thread.cpp:20] [TraceThread] coinstatsindex thread start
                                    - 2024-04-26T22:23:21.049821Z [txindex] [index/base.cpp:195] [Sync] Syncing txindex with block chain from height 0
                                    - 2024-04-26T22:23:21.147000Z [initload] [kernel/mempool_persist.cpp:73] [LoadMempool] Loading 0 mempool transactions from file...
                                    - 2024-04-26T22:23:21.155535Z [initload] [kernel/mempool_persist.cpp:145] [LoadMempool] Imported mempool transactions from file: 0 succeeded, 0 failed, 0 expired, 0 already there, 0 waiting for initial broadcast
                                    - 2024-04-26T22:23:21.336281Z [initload] [util/thread.cpp:22] [TraceThread] initload thread exit
                                    - 2024-04-26T22:23:21.344912Z [basic block filter index] [flatfile.cpp:69] [Allocate] [validation] Pre-allocating up to position 0x100000 in fltr00000.dat
                                    - 2024-04-26T22:23:21.358499Z [basic block filter index] [index/base.cpp:195] [Sync] Syncing basic block filter index with block chain from height 0
                                    - 2024-04-26T22:23:22.965573Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 0
                                    - 2024-04-26T22:23:24.073588Z [txindex] [index/base.cpp:209] [Sync] txindex is enabled at height 200
                                    - 2024-04-26T22:23:24.074095Z [txindex] [util/thread.cpp:22] [TraceThread] txindex thread exit
                                    - 2024-04-26T22:23:25.467454Z [basic block filter index] [index/base.cpp:209] [Sync] basic block filter index is enabled at height 200
                                    - 2024-04-26T22:23:25.467705Z [basic block filter index] [util/thread.cpp:22] [TraceThread] basic block filter index thread exit
                                    - 2024-04-26T22:23:53.661723Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 17
                                    - 2024-04-26T22:24:05.820494Z [scheduler] [net.cpp:2329] [StartExtraBlockRelayPeers] [net] enabling extra block-relay-only peers
                                    - 2024-04-26T22:24:24.630070Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 35
                                    - 2024-04-26T22:24:55.720035Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 54
                                    - 2024-04-26T22:25:26.457788Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 77
                                    - 2024-04-26T22:25:57.224169Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 96
                                    - 2024-04-26T22:26:28.650838Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 117
                                    - 2024-04-26T22:27:00.044150Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 137
                                    - 2024-04-26T22:27:30.385389Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 152
                                    - 2024-04-26T22:28:01.495158Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 168
                                    - 2024-04-26T22:28:31.759504Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 181
                                    - 2024-04-26T22:29:03.312048Z [coinstatsindex] [index/base.cpp:195] [Sync] Syncing coinstatsindex with block chain from height 199
                                    - 2024-04-26T22:29:05.924504Z [coinstatsindex] [index/base.cpp:209] [Sync] coinstatsindex is enabled at height 200
                                    - 2024-04-26T22:29:05.924873Z [coinstatsindex] [util/thread.cpp:22] [TraceThread] coinstatsindex thread exit
                                    - 2024-04-26T22:38:20.253388Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  202ms
                                    - 2024-04-26T22:53:20.271084Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms
                                    - 2024-04-26T23:08:20.330833Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms
                                    - 2024-04-26T23:23:11.404762Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat.
                                    - 2024-04-26T23:23:20.372298Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  16ms
                                    - 2024-04-26T23:38:20.399361Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms
                                    - 2024-04-26T23:53:20.408714Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms
                                    - 2024-04-27T00:08:20.435724Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms
                                    - 2024-04-27T00:23:11.582356Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat.
                                    - 2024-04-27T00:23:20.449989Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms
                                    - 2024-04-27T00:38:20.457850Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms
                                    - 2024-04-27T00:53:20.470015Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  7ms
                                    - 2024-04-27T01:08:20.482908Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  11ms
                                    - 2024-04-27T01:23:11.670237Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat.
                                    - 2024-04-27T01:23:20.492458Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms
                                    - 2024-04-27T01:38:20.502842Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms
                                    - 2024-04-27T01:53:20.512615Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  8ms
                                    - 2024-04-27T02:08:20.523052Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms
                                    - 2024-04-27T02:23:11.771345Z [scheduler] [policy/fees.cpp:956] [FlushFeeEstimates] Flushed fee estimates to fee_estimates.dat.
                                    - 2024-04-27T02:23:20.534687Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  10ms
                                    - 2024-04-27T02:38:20.545271Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  9ms
                                    - 2024-04-27T02:53:20.557320Z [scheduler] [net.cpp:2292] [DumpAddresses] [net] Flushed 0 addresses to peers.dat  10ms

--- a/ci/test/00_setup_env_native_valgrind.sh
+++ b/ci/test/00_setup_env_native_valgrind.sh
@@ -11,6 +11,6 @@ export CONTAINER_NAME=ci_native_valgrind
 export PACKAGES="valgrind clang-16 llvm-16 libclang-rt-16-dev python3-zmq libevent-dev libboost-dev libdb5.3++-dev libminiupnpc-dev libnatpmp-dev libzmq3-dev libsqlite3-dev"
 export USE_VALGRIND=1
 export NO_DEPENDS=1
-export TEST_RUNNER_EXTRA="--exclude rpc_bind,feature_bind_extra"  # Excluded for now, see https://github.com/bitcoin/bitcoin/issues/17765#issuecomment-602068547
+export TEST_RUNNER_EXTRA="--exclude feature_init,rpc_bind,feature_bind_extra"  # feature_init excluded for now, see https://github.com/bitcoin/bitcoin/issues/30011 ; bind tests excluded for now, see https://github.com/bitcoin/bitcoin/issues/17765#issuecomment-602068547
 export GOAL="install"
 export BITCOIN_CONFIG="--enable-zmq --with-incompatible-bdb --with-gui=no CC=clang-16 CXX=clang++-16" # TODO enable GUI

